metadata-store-directory-common/src/main/java/org/apache/helix/msdcommon/datamodel/MetadataStoreRoutingData.java

```diff
@@ -22,6 +22,7 @@
 import java.util.Map;
 import java.util.NoSuchElementException;
 
+
```

Comment:
The formatter used before was outdated. The updated formatter added the blank line.